sanic/server.py

```diff
@@ -441,7 +441,7 @@ def bail_out(self, message, from_error=False):
             logger.error("Transport closed @ %s and exception "
                          "experienced during error handling",
                          self.transport.get_extra_info('peername'))
-            logger.debug('Exception:\n%s', traceback.format_exc())
+            logger.debug('Exception:', exc_info=True)
```

Comment:
`bail_out` this is kinda a wonky function signature in general now that I'm thinking about it, if the caller supplies `from_error=True` then the function ignores the required message argument. I'm not exactly sure why we specify `from_error` or not here but I think we could simplify this by having message be an optional argument and this branch triggering if its not supplied

Reply:
not sure if this needs to be addressed in this PR either, seems like a logical local change in the context of whats already here

Reply:
I'm going to ignore this for this PR. To be honest my only objective here is to have exc_info available on messages.